package.json

```diff
@@ -71,13 +71,13 @@
     "deepmerge": "^2.0.1",
     "dom-helpers": "^3.2.1",
     "hoist-non-react-statics": "^2.3.1",
-    "jss": "^9.5.0",
```

Comment:
why did you need this? I thought it is only vendor prefixer thing

Reply:
Better keep the jss scope as wide as possible.